release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -503,6 +503,17 @@ The Network Observability Operator releases updates independently from the {prod
 [id="ocp-4-17-etcd-certificates_{context}"]
 === Security
 
+[id="ocp-17-auto-rotate-etcd-ca"]
+==== Automatic rotation of signer certificates and a new namespace
+
+With this release, all `etcd` certificates originate from a new namespace: `openshift-etcd`. When a new signer certificate is close to its expiration date, the following actions occur:
+
+* An automatic rotation of the signer certificate activates.
+* The certificate bundle updates.
+* All certificates regenerate with the new signers.
+
+Manual rotation of signer certificates is still supported by deleting the specific secret and waiting for the status pod rollout to complete.
```

Comment:
as I though, "waiting for the status pod rollout to complete" updated to "waiting for cluster operator status become to health", @tjungblu, please correct me if it's not exact, thanks

Reply:
I think it means the same :)

Reply:
ok, thanks